### DISCUSSION TITLE
openwrt router's docker man to install teslamate

### DISCUSSION THREAD

**Discussion by tokenlistio** [teslamate-org/teslamate#3372] [Q&A]
我的软路由是istore os, openwrt系统，上面有一个docker。但是这个docker只能用command 安装teslamate，请教大家如何安装？
My soft router is istore os, openwrt system, with a docker on it. But this docker can only use command to install teslamate. Could you please tell me how to install it?
![docker1](https://github.com/adriankumpf/teslamate/assets/148476462/a4470873-4674-4e3d-8eac-3009390729e5)
![docker2](https://github.com/adriankumpf/teslamate/assets/148476462/065b4ee3-3b1b-4655-b741-02ca06610dbc)

**Comment by 13311031900** [comment]
请问装上了吗？

**Reply by tokenlistio** [reply]
至今没搞好。请教兄弟你知道怎么弄吗？

**Reply by Patrickooooo** [reply]
在软件包里找到docker-compose并安装，安装之后再按教程操作。

**Reply by 13311031900** [reply]
能再具体点吗？谢谢

**Reply by 13311031900** [reply]
需要外挂硬盘吗 还是本身自带的硬盘容量就够了 我得是rs4s

**Reply by Patrickooooo** [reply]
我是x86的，ssh登陆istore之后按照教程安装，不过istore没有安装docker-compose，自己在软件包里安装一下，后面按照教程。不知道你硬盘多大，这是运行了一年半的容器存储情况。
<img width="199" alt="截屏2024-01-29 11 03 39" src="https://github.com/teslamate-org/teslamate/assets/157789519/a0cb038f-0fdb-4058-8057-9491cc38b744">
<img width="473" alt="截屏2024-01-29 11 05 23" src="https://github.com/teslamate-org/teslamate/assets/157789519/e217f74e-fd44-4d2e-b42f-d1d448bac7bb">

**Comment by zhoubingyi** [comment]
我倒是能装上
可是有两个服务起不起来，不知道为什么
最后用了电脑。。。

**Reply by lixz123007** [reply]
中文版的teslamate只提供amd64。也就是x86。arm64和v7的镜像他们没转。想到树莓派上就得用teslamate的源。

**Reply by zhoubingyi** [reply]
是的，官方的在 arm 上安装成功了

**Comment by belugaming** [comment]
> 我的软路由是istore os, openwrt系统，上面有一个docker。但是这个docker只能用command 安装teslamate，请教大家如何安装？
> 
> My soft router is istore os, openwrt system, with a docker on it. But this docker can only use command to install teslamate. Could you please tell me how to install it?
> 
> ![docker1](https://github.com/adriankumpf/teslamate/assets/148476462/a4470873-4674-4e3d-8eac-3009390729e5)
> 
> ![docker2](https://github.com/adriankumpf/teslamate/assets/148476462/065b4ee3-3b1b-4655-b741-02ca06610dbc)
> 
> 

我装好了，用compose安装的，需要软件包安装docker-compose 然后进ssh安装jaml文件

**Reply by lesucret** [reply]
你安装好了key能验证通过吗

**Reply by belugaming** [reply]
> 你安装好了key能验证通过吗

当然可以

**Reply by zhoubingyi** [reply]
我的是RS5刷的istore，可以安装正常使用

**Comment by lesucret** [comment]
我的是x86，一直验证失败。在 2024年6月2日，13:09，beluga ming ***@***.***> 写道：﻿

你安装好了key能验证通过吗

当然可以

**Reply by belugaming** [reply]
> 我的是x86，一直验证失败。在 2024年6月2日，13:09，beluga ming ***@***.***> 写道：﻿ 你安装好了key能验证通过吗 当然可以 —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

网络模式全改为host模式 或者你看看是不是密钥填反了 掉换一下看看

**Comment by lesucret** [comment]
我的是x86安装istore安装了，key每次都说失效。发自我的 iPhone在 2024年6月5日，20:15，momo ***@***.***> 写道：﻿
我的是RS5刷的istore，可以安装正常使用
